app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  validates :content, presence: true
```

Comment:
There and in other models 
https://github.com/arbox/rails-style-guide/blob/master/README-ruRU.md#macro-style-methods